fdbclient/RESTUtils.actor.cpp

```diff
@@ -281,7 +308,17 @@ TEST_CASE("/RESTUtils/ValidURIWithoutService") {
 	ASSERT_EQ(r.connType.secure, RESTConnectionType::SECURE_CONNECTION);
 	ASSERT_EQ(r.host.compare("host"), 0);
 	ASSERT(r.service.empty());
-	ASSERT_EQ(r.resource.compare("foo/bar"), 0);
+	ASSERT_EQ(r.resource.compare("/foo/bar"), 0);
+	return Void();
+}
+
+TEST_CASE("/RESTUtils/ValidURIWithExtraForwardSlash") {
+	std::string uri("https://host//foo/bar");
+	RESTUrl r(uri);
+	ASSERT_EQ(r.connType.secure, RESTConnectionType::SECURE_CONNECTION);
+	ASSERT_EQ(r.host.compare("host"), 0);
+	ASSERT(r.service.empty());
+	ASSERT_EQ(r.resource.compare("/foo/bar"), 0);
```

Comment:
Do you want to apply the fix that was merged on main to this PR?

Reply:
Otherwise as soon as this is merged we will start seeing sporadic failures of CI

Reply:
Sure, I was in double mind with it, but better to apply here. First, was trying to keep parity of checkins between branches, but I guess that doesn't matter.